js_modules/dagster-ui/packages/app-oss/package.json

```diff
@@ -37,13 +37,17 @@
     "@types/uuid": "^8.3.4",
     "@types/validator": "^13",
     "@types/webpack-bundle-analyzer": "^4",
+    "@typescript-eslint/eslint-plugin": "^8.9.0",
+    "@typescript-eslint/parser": "^8.9.0",
     "eslint": "^8.57.0",
+    "eslint-plugin-import": "^2.31.0",
     "eslint-plugin-jest": "^26.4.6",
     "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-unused-imports": "^4.1.4",
```

Comment:
These started showing up as missing during lint runs, since they were only in the eslint-config's node_modules. I'm not sure why this worked without enumerating these deps before, but I think we were just getting lucky with packages ending up in the workspace-level node_modules.